tests/ttnn/unit_tests/operations/eltwise/test_pow.py

```diff
@@ -182,3 +182,73 @@ def test_binary_sfpu_pow_neg(
 
     pcc = ttnn.pearson_correlation_coefficient(torch_output_tensor, output)
     assert pcc >= 0.99
+
+
+@skip_for_grayskull("Unsupported dtype for Grayskull")
+@pytest.mark.parametrize(
+    "ttnn_function",
+    [
+        ttnn.pow,
+    ],
+)
+def test_pow_fp32(device, ttnn_function):
+    x_torch = torch.tensor([[0.98828125, 0.47851562, 1.1875, -1.59375]], dtype=torch.float32)
+    y_torch = torch.tensor([[0.0751953125, 0.53125, -0.6640625, 0.1533203125]], dtype=torch.float32)
+    golden_fn = ttnn.get_golden_function(ttnn_function)
+    z_torch = golden_fn(x_torch, y_torch)
+    x_tt = ttnn.from_torch(x_torch, dtype=ttnn.float32, layout=ttnn.TILE_LAYOUT, device=device)
+    y_tt = ttnn.from_torch(y_torch, dtype=ttnn.float32, layout=ttnn.TILE_LAYOUT, device=device)
+    z_tt_pow = ttnn.pow(x_tt, y_tt)
+    tt_out = ttnn.to_torch(z_tt_pow)
+    # z_tt_pow ttnn.Tensor([[ 0.99930,  0.68274,  ...,  0.90147,      nan]])
+    # z_torch tensor([[0.9991, 0.6760, 0.8922,    nan]])
+    status = ttnn.pearson_correlation_coefficient(z_torch, tt_out) >= 0.99
+    assert status
+
+
+@skip_for_grayskull("Unsupported dtype for Grayskull")
+@pytest.mark.parametrize(
+    "ttnn_function",
+    [
+        ttnn.pow,
+    ],
+)
+def test_pow_bf16(device, ttnn_function):
+    x_torch = torch.tensor([[0.98828125, 0.478515625, 1.1875, -1.59375]], dtype=torch.bfloat16)
+    y_torch = torch.tensor([[0.0751953125, 0.53125, -0.6640625, 0.1533203125]], dtype=torch.bfloat16)
+    golden_fn = ttnn.get_golden_function(ttnn_function)
+    z_torch = golden_fn(x_torch, y_torch)
+    x_tt = ttnn.from_torch(x_torch, dtype=ttnn.bfloat16, layout=ttnn.TILE_LAYOUT, device=device)
+    y_tt = ttnn.from_torch(y_torch, dtype=ttnn.bfloat16, layout=ttnn.TILE_LAYOUT, device=device)
+    z_tt_pow = ttnn.pow(x_tt, y_tt)
+    tt_out = ttnn.to_torch(z_tt_pow)
+    # Due to HW limitations for bfloat16 dtype, NaN value gets packed as inf.
+    # z_tt_pow ttnn.Tensor([[ 0.99609,  0.67969,  ...,  0.89844,      inf]])
+    # z_torch tensor([[1.0000, 0.6758, 0.8906,    nan]], dtype=torch.bfloat16)
+
+    status = ttnn.pearson_correlation_coefficient(z_torch, tt_out) >= 0.99
+    assert status
```

Comment:
Was it intentional to `skip_for_grayskull` on bf16? If bf16 is supposed to run on Grayskull, then do this:
```suggestion
@pytest.mark.parametrize(
    "dtype",
    [
        "float32",
        "bfloat16",
    ],
)
def test_binary_pow(device, dtype):
    if dtype == "float32" and is_grayskull():
        pytest.skip("Unsupported dtype for Grayskull")

    torch_dtype = getattr(torch, dtype)
    ttnn_dtype = getattr(ttnn, dtype)
    x_torch = torch.tensor([[0.98828125, 0.47851562, 1.1875, -1.59375]], dtype=torch_dtype)
    y_torch = torch.tensor([[0.0751953125, 0.53125, -0.6640625, 0.1533203125]], dtype=torch_dtype)
    z_torch = torch.pow(x_torch, y_torch)
    x_tt = ttnn.from_torch(x_torch, dtype=ttnn_dtype, layout=ttnn.TILE_LAYOUT, device=device)
    y_tt = ttnn.from_torch(y_torch, dtype=ttnn_dtype, layout=ttnn.TILE_LAYOUT, device=device)
    z_tt_pow = ttnn.pow(x_tt, y_tt)
    tt_out = ttnn.to_torch(z_tt_pow)

    status = ttnn.pearson_correlation_coefficient(z_torch, tt_out) >= 0.99
    assert status
```
otherwise just use `@skip_for_grayskull` decorator to skip for both dtypes.

Reply:
The sfpu ops are available only for wormhole and blackhole 
https://github.com/tenstorrent/tt-metal/pull/15782/files

Reply:
I have updated the tests with both dtypes